<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetParameters</class>
 <widget class="QMainWindow" name="SetParameters">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>997</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Set Parameters</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="camera_video">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>741</width>
      <height>451</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">Background-color: #000;</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="initCapture">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>330</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Off</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="fps_number">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>380</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">Background-Color: #000</string>
    </property>
   </widget>
   <widget class="QLabel" name="fps_label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>380</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>FPS:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="readParameters">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Read Parameters</string>
    </property>
   </widget>
   <widget class="QPushButton" name="configRobots">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Robots</string>
    </property>
   </widget>
   <widget class="QPushButton" name="T1_color">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Team 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="T2_color">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Team 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mapPoints">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Map</string>
    </property>
   </widget>
   <widget class="QPushButton" name="attackArrea_btn">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>230</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Attack Area</string>
    </property>
   </widget>
   <widget class="QPushButton" name="defenseArrea_btn">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>270</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Defense Area</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ball_color">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Ball</string>
    </property>
   </widget>
   <widget class="QLabel" name="camera_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Camera:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>380</y>
      <width>47</width>
      <height>23</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="value">
     <number>300</number>
    </property>
   </widget>
   <widget class="QPushButton" name="config_serial">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>120</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Serial</string>
    </property>
   </widget>
   <widget class="QPushButton" name="calibrate_camera">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Calibrate (Video4Linux)</string>
    </property>
   </widget>
   <widget class="QLabel" name="setColorlbl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Colors:</string>
    </property>
   </widget>
   <widget class="QLabel" name="configurelbl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Configure:</string>
    </property>
   </widget>
   <widget class="QLabel" name="setPointslbl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Points:</string>
    </property>
   </widget>
   <widget class="QLabel" name="idlbl">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>67</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ID:</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
